Dockerfile

```diff
@@ -42,7 +42,9 @@ RUN if echo "$COMPUTE_BACKENDS" | grep -q "slurmcern"; then \
                           --no-install-recommends; \
     fi
 
-ADD etc/cernsubmit.yaml /etc/condor/
+RUN mkdir -p /etc/myschedd 
```

Comment:
Please delete trailing whitespace after this line.

Reply:
Fixed!